2025 song catalog

challfry opened the pull request:
~600 new songs for 2025.

cohoe commented:
Are you concerned about the unicode-y-ness of ‘ vs ' ? There's a Python script I wrote in the prod repo that I used to generate the seed in there from that spreadsheet we got. Didn't backfill the seed here.

challfry commented:
The file showed up with different places in the file using either `ë` or `í` characters for single quotes. This may be because it had legacy Mac line endings, which made BBEdit assume an encoding of Mac OS Roman instead of UTF-8. 

I fixed the line endings, changed the file to tab-delimited, and removed the double quotes around each field. I then converted the single quotes. Every step in our chain should be UTF-8, so the quotes should be fine.
